src/Areas/AzureArc/Options/ArcConnectOptions.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace AzureMcp.Options.Arc;
+
+public class ArcConnectOptions : GlobalOptions
```

Comment:
Resource group and tenant are already in GlobalOptions.

You can use SubscriptionOptions which also includes Subscription.